packages/@react-spectrum/provider/test/Provider.test.js

```diff
@@ -193,8 +202,61 @@ describe('Provider', () => {
     act(() => {
       matchMedia.useMediaQuery(mediaQueryDark);
     });
-    
+
     expect(provider1.classList.contains('spectrum--dark')).toBeTruthy();
     expect(provider2.classList.contains('spectrum--dark')).toBeTruthy();
   });
+
+  describe('responsive styles', function () {
+    let breakpoints = {S: 480, M: 640, L: 1024};
+    // jsdom/cssstyle doesn't support var() yet, so we need to use other values
```

Comment:
Potentially we can go back to vars if this gets merged https://github.com/jsdom/cssstyle/pull/127